recipes/paho-mqtt-c/all/conanfile.py

```diff
@@ -88,7 +88,10 @@ def package_info(self):
             elif self.settings.os == "FreeBSD":
                 self.cpp_info.system_libs.extend(["compat", "pthread"])
             else:
-                self.cpp_info.system_libs.extend(["c", "pthread"])
+                if self.settings.os == "Android":
```

Comment:
Could you please add this as another `elsif` clause? I think it is easier to read

Reply:
sure, will commit soon
and btw, I do somehow not feel responsible for the failed build for the old version.... this should not happen by this change